documentation/source/users/arkane/input.rst

```diff
@@ -863,7 +863,22 @@ Use a ``kinetics()`` function to make Arkane execute the high-pressure limit kin
 parameters computation for a reaction. The ``'label'`` string must correspond to that of
 a defined ``reaction()`` function.
 
-You have three options for specifying the temperature to which a modified Arrhenius
+By default, Arkane uses a modified three-parameter Arrhenius equation to fit kinetics data:
```

Comment:
suggestion:
"By default, Arkane outputs high pressure kinetic rate coefficients in the modified three-parameter Arrhenius equation format:"

Reply:
ok